metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -61,47 +62,60 @@ final class Embedded(
 }
 
 object Embedded {
-  def downloadSettings(
-      dependency: Dependency,
-      scalaVersion: String
-  ): Settings =
-    new coursiersmall.Settings()
-      .withTtl(Some(Duration.Inf))
-      .withDependencies(List(dependency))
-      .addRepositories(
-        List(
-          coursiersmall.Repository.SonatypeReleases,
-          coursiersmall.Repository.SonatypeSnapshots
+  lazy val repositories: mutable.Buffer[Repository] =
```

Comment:
That shouldn't really be a mutable buffer though (looks really bad). Do a `toList` on it.

Reply:
Fixed.